### PR TITLE
New package: atrenton.MyJournal.Notebook version 16.4.0

### PR DESCRIPTION
**Pull request by atrenton**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121043)

### CODE REVIEW

**Comment by atrenton** [comment]
@microsoft-github-policy-service agree

**Comment by atrenton** [comment]
The installation validation failed as the package checks that Microsoft OneNote 2016 or later is installed as a prerequisite. I ran 'WinGet install' on my PC and it worked fine, as I have OneNot 2016 installed.

**Comment by atrenton** [comment]
So, after doing some research it appears that package dependencies are currently in experimental support.

-  [#163](https://github.com/microsoft/winget-cli/issues/163)
-  [#163 - Dependencies.md](https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23163%20-%20Dependencies.md) spec

Please advise on how to proceed. I've been able to locate my dependencies using the following queries:

- winget search --id Microsoft.DotNet.DesktopRuntime.6
- winget search --name OneNote --id XPFFZHVGQWWLHB

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Installation failed with exit code -1978335226 
2023-09-27 18:01:12.375 [CLI ] ShellExecute installer failed: 1

**Comment by stephengillie** [comment]
Manual Validation ended withL
image

Should the `msstore` package "Microsoft 365 (Office)" `9WZDNCRD29V9` be a dependency?

**Comment by atrenton** [comment]
Attempting to use OneNote as a dependency doesn't work.  Submitted a **winget-cli** issue:
Microsoft OneNote PackageIdentifier value fails schema validation [#3720](https://github.com/microsoft/winget-cli/issues/3720)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2023-10-03 17:53:00.684 [CLI ] ShellExecute installer failed: 1 
2023-10-03T17:53:00]e000: Error 0x81f40001: Bundle condition evaluated to false: Installed OR ON16_32 OR ON16_64

**Comment by stephengillie** [comment]
> So, after doing some research it appears that package dependencies are currently in experimental support.
> 
> * [#163](https://github.com/microsoft/winget-cli/issues/163)
> * [#163 - Dependencies.md](https://github.com/microsoft/winget-cli/blob/master/doc/specs/%23163%20-%20Dependencies.md) spec
> 
> Please advise on how to proceed. I've been able to locate my dependencies using the following queries:
> 
> * winget search --id Microsoft.DotNet.DesktopRuntime.6
> * winget search --name OneNote --id XPFFZHVGQWWLHB

Hi @atrenton,

I missed this reply and misread your other reply, sorry.

File: manifests/a/atrenton/MyJournal/Notebook/16.4.0/atrenton.MyJournal.Notebook.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: atrenton.MyJournal.Notebook
+PackageVersion: 16.4.0
+InstallerType: burn
+Installers:
```
Comment (stephengillie) [inline manifests/a/atrenton/MyJournal/Notebook/16.4.0/atrenton.MyJournal.Notebook.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
  - PackageIdentifier: Microsoft.Office
Installers:
```

Comment (stephengillie) [reply manifests/a/atrenton/MyJournal/Notebook/16.4.0/atrenton.MyJournal.Notebook.installer.yaml:7]: I believe that `Microsoft.Office` contains `OneNote`.

**Comment by stephengillie** [comment]
-  Manifest Error: Schema Error. [Schema validation failed. 
  1. Error context: <root>[Installers][0][Dependencies][PackageDependencies][1][PackageIdentifier] Description: Failed to match regex specified by 'pattern' constraint. 
  2. Error context: <root>[Installers][0][Dependencies][PackageDependencies][1] Description: Failed to validate against schema associated with property name 'PackageIdentifier'. 
  3. Error context: <root>[Installers][0][Dependencies][PackageDependencies] Description: Failed to validate item 1 in array. Error context: <root>[Installers][0][Dependencies] Description: Failed to validate against schema associated with property name 'PackageDependencies'. 
  4. Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'Dependencies'. 
  5. Error context: <root>[Installers] Description: Failed to validate item #0 in array. 
  6. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. 
- ] File: atrenton.MyJournal.Notebook.installer.yaml

**Comment by stephengillie** [comment]
The issue seems to be a new issue - the install is blocked because there are no OneNote (XPFFZHVGQWWLHB) installers which match the main package's scope and architecture: 

Manual Validation ended with:
```
2023-10-04 15:51:20.803 [REPO] Sending http GET request to: https://storeedgefd.dsx.mp.microsoft.com/v9.0/packageManifests/XPFFZHVGQWWLHB?Market=US
2023-10-04 15:51:20.907 [REPO] Response status: 200
2023-10-04 15:51:21.234 [REPO] Finding installed package from available package using system reference search: Query:[none] Include:ProductCode='onenotefreeretail - ar-sa'[Exact] Include:ProductCode='onenotefreeretail - bg-bg'[Exact] Include:ProductCode='onenotefreeretail - cs-cz'[Exact] Include:ProductCode='onenotefreeretail - da-dk'[Exact] Include:ProductCode='onenotefreeretail - de-de'[Exact] Include:ProductCode='onenotefreeretail - el-gr'[Exact] Include:ProductCode='onenotefreeretail - en-gb'[Exact] Include:ProductCode='onenotefreeretail - en-us'[Exact] Include:ProductCode='onenotefreeretail - es-es'[Exact] Include:ProductCode='onenotefreeretail - es-mx'[Exact] Include:ProductCode='onenotefreeretail - et-ee'[Exact] Include:ProductCode='onenotefreeretail - fi-fi'[Exact] Include:ProductCode='onenotefreeretail - fr-fr'[Exact] Include:ProductCode='onenotefreeretail - he-il'[Exact] Include:ProductCode='onenotefreeretail - hi-in'[Exact] Include:ProductCode='onenotefreeretail - hr-hr'[Exact] Include:ProductCode='onenotefreeretail - hu-hu'[Exact] Include:ProductCode='onenotefreeretail - id-id'[Exact] Include:ProductCode='onenotefreeretail - it-it'[Exact] Include:ProductCode='onenotefreeretail - ja-jp'[Exact] Include:ProductCode='onenotefreeretail - kk-kz'[Exact] Include:ProductCode='onenotefreeretail - ko-kr'[Exact] Include:ProductCode='onenotefreeretail - lt-lt'[Exact] Include:ProductCode='onenotefreeretail - lv-lv'[Exact] Include:ProductCode='onenotefreeretail - ms-my'[Exact] Include:ProductCode='onenotefreeretail - nb-no'[Exact] Include:ProductCode='onenotefreeretail - nl-nl'[Exact] Include:ProductCode='onenotefreeretail - pl-pl'[Exact] Include:ProductCode='onenotefreeretail - pt-br'[Exact] Include:ProductCode='onenotefreeretail - pt-pt'[Exact] Include:ProductCode='onenotefreeretail - ro-ro'[Exact] Include:ProductCode='onenotefreeretail - ru-ru'[Exact] Include:ProductCode='onenotefreeretail - sk-sk'[Exact] Include:ProductCode='onenotefreeretail - sl-si'[Exact] Include:ProductCode='onenotefreeretail - sr-latn-rs'[Exact] Include:ProductCode='onenotefreeretail - sv-se'[Exact] Include:ProductCode='onenotefreeretail - th-th'[Exact] Include:ProductCode='onenotefreeretail - tr-tr'[Exact] Include:ProductCode='onenotefreeretail - uk-ua'[Exact] Include:ProductCode='onenotefreeretail - vi-vn'[Exact] Include:ProductCode='onenotefreeretail - zh-cn'[Exact] Include:ProductCode='onenotefreeretail - zh-tw'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - ar-sa'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - bg-bg'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - cs-cz'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - da-dk'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - de-de'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - el-gr'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - en-gb'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - en-us'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - es-es'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - es-mx'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - et-ee'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - fi-fi'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - fr-fr'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - he-il'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - hi-in'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - hr-hr'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - hu-hu'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - id-id'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - it-it'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - ja-jp'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - kk-kz'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - ko-kr'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - lv-lv'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - ms-my'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - nb-no'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - nl-nl'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - pl-pl'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - pt-br'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - pt-pt'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - ro-ro'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - ru-ru'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - sk-sk'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - sl-si'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - sr-latn-rs'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - sv-se'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - th-th'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - tr-tr'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - uk-ua'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - vi-vn'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - zh-cn'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='microsoft onenote - zh-tw'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='onenote'+'microsoft corporation'[Exact] Include:NormalizedNameAndPublisher='„microsoft onenote“ - lt-lt'+'microsoft corporation'[Exact]
2023-10-04 15:51:21.264 [REPO] Opening pinning index
2023-10-04 15:51:21.264 [REPO] Opening existing pinning index
2023-10-04 15:51:21.264 [REPO] Opening SQLite Index for ReadWrite at 'C:\Users\User\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\pinning.db'
2023-10-04 15:51:21.264 [SQL ] Opening SQLite connection #6: 'C:\Users\User\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\pinning.db' [2, 0]
2023-10-04 15:51:21.264 [REPO] Opened Pinning Index with version [1.0], last write [2023-06-28 12:54:26.000]
2023-10-04 15:51:21.267 [CLI ] Starting installer selection.
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,af] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,sq] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,am] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,hy] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,as] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,az] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,bn] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,eu] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,be] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,bs] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ca] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,en] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,en-us] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,fil] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,gl] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ka] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,gu] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,is] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ga] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,kn] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,km] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,lb] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,mk] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ml] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,mt] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,mi] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,mr] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ne] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,or] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,fa] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,quz] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ta] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,tt] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,te] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ur] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ug] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,uz] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ca-es-valencia] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,cy] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,nn] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,nn-no] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,no] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,zh-hans] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,de] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,es] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,es-es] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,fr] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,it] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ja] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,nl] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,pt-br] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ru] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,da] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,fi] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ko] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,nb] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,nb-no] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,pl] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,sv] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,tr] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,zh-hant] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,ar] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,cs] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,el] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,he] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,hu] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.267 [CLI ] Installer [Neutral,exe,Machine,sk] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,th] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,bg] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,et] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,hr] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,id] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,lt] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,lv] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,ro] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,sl] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,sr] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,uk] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,vi] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,hi] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,pa] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,pt-pt] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,pt] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,ms] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,kk] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,en-gb] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.268 [CLI ] Installer [Neutral,exe,Machine,es-mx] not applicable: Architecture was excluded by caller : Neutral
2023-10-04 15:51:21.271 [CLI ] No suitable installer found for manifest XPFFZHVGQWWLHB with version 16.0.16501.20228
2023-10-04 15:51:21.273 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:11a
```

**Pull request closed without merging**